examples/kubernetes/steps/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .evaluator import evaluator
```

Comment:
.evaluator? Im not super used to thta notation. why is the dot neccessary?

Reply:
It's a relative import, e.g. `from .evaluator import ...` is the same as calling `from steps.evaluator import ...` (if your CWD is in `examples/kubernetes`).

I noticed our style guide recommends against relative imports, so I changed them all to absolute now.